### DISCUSSION TITLE
How to change the default ripple setting of all components

### DISCUSSION THREAD

**Discussion by kingyue737** [quasarframework/quasar#12795] [General - Components / Directives / etc] [answered]
As mentioned in #6524, ripple triggered by mouseup instead of mousedown is a very strange behavior. Is it possible to set mousedown as default instead of manually adding ripple props in all the components?

**Comment by yusufkandemir** [comment, marked as answer]
https://quasar.dev/vue-directives/material-ripple#ripple-api
![image](https://user-images.githubusercontent.com/6266078/158334922-ba61468f-012f-4f2f-81d2-dc37dd0bdbf2.png)
